core/storage/statistics/gae_models.py

```diff
@@ -1237,37 +1236,6 @@ def get_multi_stats_models(cls, exp_version_references):
         exploration_stats_models = cls.get_multi(entity_ids)
         return exploration_stats_models
 
-    @classmethod
-    def save_multi(cls, exploration_stats_dicts):
```

Comment:
Why is this deleted?

Reply:
This (and all other deleted methods) are unused code after deleting the jobs.

Reply:
Hm I think this one is OK, but the ones below (create_or_update etc.) mean that the corresponding model doesn't have any useful methods on it any more.

Maybe this is OK but it might be worth adding a breadcrumb (for classes like those) to say that "NOTE TO DEVELOPERS: The methods create_or_update(), XXX(), YYY() were removed in #13021 as part of the migration to Apache Beam. Please refer to that PR if you need to reinstate them".

Reply:
Done.